packages/pluggableWidgets/rich-text-web/src/RichText.editorPreview.tsx

```diff
@@ -0,0 +1 @@
+export const preview = (): null => null;
```

Comment:
Question: 
- Was it intentional to remove the entire editor preview?
- If so, what is the plan for this in the future?

Reply:
It's the same case as in Charts, we need SVG